### ISSUE TITLE
NSTStreetView Does not exist

### ISSUE THREAD

**Issue by shikya** [nesterapp/react-native-streetview#5]
I followed all steps to set up Google Maps and Places.
`ExceptionsManager.js:71 Warning: Native component for "NSTStreetView" does not exist`

**Comment by jsappme** [comment]
same issue here, no solution?

**Comment by zsajjad** [comment]
You might need to add.

```java
return Arrays.<ReactPackage>asList(
      ...  
      new NSTStreetViewPackage(),
       ...
      );
```

in `MainApplication.java`  as `react-native link` is not adding this.

**Comment by lgngnr** [comment]
react-native link do this but still same issue...

**Comment by scotbond** [comment]
solved in 0.2.1
